README.md

```diff
@@ -95,12 +95,7 @@ In order to run the events warehouse locally, you will need to:
 - configure local databases: run `bundle exec rails db:schema:load`;
 - seed local databases: run `bundle exec rails db:seed`;
 - have a RabbitMQ server running locally: this can be done through _brew services_;
-- start the consumer: in a terminal run `bundle exec ./bin/amqp_client start`;
-- create a new queue: the queue name should be as specified in the `development.rb` config file (e.g. `config.amqp.main.queue = 'queue'`);
-- bind an exhange to the new queue (with routing keys if appropriate): the exchange should be the one your publisher will write to.
-
-Any amqp processes can be stopped by running `bundle exec ./bin/amqp_client stop`.
-
```

Comment:
This has got a whole lot simpler, as the consumers:
- Are no longer daemonized, so just run in the foreground
- Register their own queues

Reply:
How can you stop it now? Ctrl-C if you're running it on your machine I guess? How about on the server - kill the process?

Reply:
Yep, ctrl-c on your machine. 

On the server its running under systemd, which similarly sends a term signal when it shuts it down (If you try to kill the process yourself, systemd will think its crashed and restart it). Essentially this means the means of starting on stopping it on the server are unchanged. I usually use the m/monit web interface if I want to stop a process. That way it gets unmonitored as well.

https://github.com/sanger/deployment/blob/master/roles/warren/templates/warren_client.service.j2

Is the warren configuration. By setting `Type=exec` we provide systemd with all the information it needs to stop the process.

I think the command to tell systemd to shut down the process is something like:
`sudo systemctl stop APPLICATION_warren_client.service` and this is what monit triggers. (It also now uses systemxd to tell if the process is running, removing the need for a pid file)